scripts/ethereum-dev-node.ts

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env -S yarn node -r ts-node/register/transpile-only
+
+import * as fs from "fs/promises";
+import execa from "execa";
+import { deployAll } from "radicle-contracts";
+import * as ethers from "ethers";
+import waitOn from "wait-on";
+
+const ethAccountFile = "sandbox/.local-eth-account";
+
+main().catch(e => {
+  console.error(e);
+  process.exit(1);
+});
```

Comment:
Do we still need to do this?

Reply:
Probably to kill all worker threads? :shrug:

Reply:
If we don’t do this an “UnhandledPromiseRejectionWarning” will be emitted which does not kill the process. This is the behavior of Node v14 which we are using in `.tool-version` and CI. Once we adopt Node v15 we can get rid of this.